.github/workflows/pullpreview.yml

```diff
@@ -17,6 +17,14 @@ on:
         description: The name of an Artifact which will be extracted while preserving file paths.
         required: false
         type: string
+      DNS:
+        description: A custom domain which will be used for the preview environment.
+        required: false
+        type: string
+      MAX_DOMAIN_LENGTH:
+        description: The maximum length which will be used for the fully qualified domain name.
```

Comment:
Maybe add here or below what exactly this does. Like, it cuts off the subdomain name to fit into this max length.

Reply:
Done.